src/data/data.js

```diff
@@ -0,0 +1,40 @@
+
+const cityCardData = {
+    name: "London",
+    count: "5102",
```

Comment:
I can see why you've declared the `count` property in `CityCardProps` as a string. It just makes more sense for the `count` property to be a number. So just use a number here instead of this string